pkgs/applications/blockchains/chia/default.nix

```diff
@@ -14,6 +18,12 @@ python3Packages.buildPythonApplication rec {
   patches = [
     # tweak version requirements to what's available in Nixpkgs
     ./dependencies.patch
+    # Allow later websockets release, https://github.com/Chia-Network/chia-blockchain/pull/6304
+    (fetchpatch {
+      name = "later-websockets.patch";
+      url = "https://github.com/Chia-Network/chia-blockchain/commit/a188f161bf15a30e8e2efc5eec824e53e2a98a5b.patch";
+      sha256 = "0rws7fhxmca3d5w0bkygx8scvzah3yvb3yfhn05qmp73mn3pmcb1";
+    })
```

Comment:
something is wrong with this patch, I get
```
error: hash mismatch in fixed-output derivation '/nix/store/5fg0vkxg3r5awc2gw338k23lfp4hn4r4-later-websockets.patch.drv':
         specified: sha256-YbF6h63j3IoLsND5sbYfUP3NNOrPzwV4aUOx2qE7mmc=
            got:    sha256-Gp0HwOGXHV2rxrbyZTGCu3+WgC3snmjNRyLWSRqUuOg=
error: 1 dependencies of derivation '/nix/store/a8yw9l461g6jia37dc2lsfn3s54qp7p1-chia-1.1.7.drv' failed to build
```

Reply:
Fixed, thanks.